contracts/upgradeable_contracts/amb_erc677_to_erc677/BasicAMBErc677ToErc677.sol

```diff
@@ -67,6 +67,15 @@ contract BasicAMBErc677ToErc677 is
         return isInitialized();
     }
 
+    function getReceiver(address _from, bytes _data) internal view returns (address recipient) {
```

Comment:
is the name `chooseReceiver` is more meaningful?

Reply:
Thanks, yes, I think it's better https://github.com/poanetwork/poa-bridge-contracts/pull/301/commits/6601874ea3b95ede5b545c384217ae547cc98347